uniswap_swap_detection/src/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { ethers } from "forta-agent";
+import {
+  UNISWAP_FACTORY_ADDRESS,
+  UNISWAP_POOL_INIT_CODE_HASH,
+} from "./constant";
+
+export const uniswapPoolAddress = (
```

Comment:
The name of this function is confusing, because it is names like a variable. Would suggest to rename to `getUniswapPoolAddress` or something along those lines.